### ISSUE TITLE
Threaded mode

### ISSUE THREAD

**Issue by OddBloke** [hometown-fork/hometown#55]
### Pitch

TL;DR: [glitch's threaded mode](https://glitch-soc.github.io/docs/features/threaded-mode/)

This would add a UI element to the compose UI that would enable "threaded mode". When a post is posted, the compose UI is immediately setup to reply to that post.

(This is _similar_ to [Twitter's thread creation](https://help.twitter.com/en/using-twitter/create-a-thread), except this proposal does not hold the posts until the thread is complete as Twitter does.)

### Motivation

I most notice the lack of this feature when I'm scrolled down in my timeline (and want to retain that position) and want to send a sequence of posts. After I send the first one, I am left with the choice of either hitting Home (and losing my position), or opening up my profile (or timeline) in a separate tab and continuing my thread from there. This is clunky.

The two scenarios where I most run into this: I've just boosted something and want to share my (lengthy, apparently) thoughts on it before resuming my scrolling, or I'm scrolling my timeline while watching something that I want to comment on (whilst keeping all of my posts about DS9 S5E27 or whatever in a single thread).

**Comment by OddBloke** [comment]
Oh, I meant to say! I'd be happy to attempt an implementation of this if it's of interest.
